IFC4x3/Properties/s/SignalSightingRouteIndicator_1UzmZKZgT3_egnwLmHjJWR/Documentation.md

```diff
@@ -0,0 +1 @@
+Required reading distance for the route indicator.
```

Comment:
This description is the same for `SignalSightingRequiredForIndicator`.

Reply:
That one is removed https://github.com/bSI-InfraRoom/IFC-Specification/pull/179/commits/3471cf4fba3a9585bdeb8395780ec9a674ace749

Reply:
Okay, removed the other one.